src/config/config.ts

```diff
@@ -114,6 +114,7 @@ export type FlashMessageType = 'error' | 'info' | 'warning' | 'success';
 export interface FlashMessage {
     type: FlashMessageType;
     message: string;
+    title: string;
```

Comment:
do all flash messages return a title in the API? could they ever be null?

Reply:
yes, at least  a string `''`. As you can see it's the default value.
https://github.com/bigcommerce/bigcommerce/pull/37501/files#diff-49465e72c7fe395e75d0f7d26783870aecddd0fc85133dd55cf306d975a6c355R176-R190